module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php

```diff
@@ -212,11 +273,7 @@ public function loadPickUpLocations()
         } elseif ($this->config['Catalog']['pickupLocationsFromNCIP'] ?? false) {
             $this->loadPickUpLocationsFromNcip();
         } else {
-            throw new ILSException(
-                'XCNCIP2 ILS driver bad configuration. You should set up ' .
-                'one of these options: "pickupLocationsFile" or ' .
-                '"pickupLocationsFromNCIP"'
-            );
+            $this->pickupLocations = [];
```

Comment:
Is there a reason to remove this exception? Is it no longer true that pickup location configuration is necessary? If we do change this behavior, should we at least adjust some notes in XCNCIP2.ini?

Reply:
Yes, there is reason - there are many systems, without any location defined either and still accepting requests. I updated comment in ini file a bit

Reply:
Looks good!